frontend/src/pages/admin/Template/sections/Environment.vue

```diff
@@ -155,11 +157,7 @@ export default {
                         field.policy = false
                     }
                     if (field.policy === undefined && this.envVarNames[field.name] !== i) {
-                        field.error = 'Field has duplicate name'
```

Comment:
I'm not sure I follow the reason behind this change, why do we no longer need to check for duplicate ENV keys or empty names?